esp32/src/spi0.rs

```diff
@@ -205,85 +190,16 @@ impl RegisterBlock {
     pub const fn slv_rd_bit(&self) -> &SLV_RD_BIT {
         &self.slv_rd_bit
     }
-    #[doc = "0x80 - "]
+    #[doc = "0x80..0xc0 - "]
     #[inline(always)]
-    pub const fn w0(&self) -> &W0 {
-        &self.w0
+    pub const fn w(&self, n: usize) -> &W {
```

Comment:
@burrbull is there a way in svd2rust to treat a cluster of registers as slice? In the case of the W register here and many others, we often cast the first to a pointer and create a slice from that, maybe svd2rust can do this for us?

Reply:
P.S. Looks like previous example could use iterator.